storage.py

```diff
@@ -14,7 +14,7 @@ class Storage:
         Stores argument `character` if logic is met
     """
 
-    __operators: list = ['^', '/', '*', '+', '-']
+    __operators: list = ['x!','^', '/', '*', '+', '-']
```

Comment:
Don't mix operand `x` and operator `x`. Here `__operator` list contains only operators.